documentation/guide-how-to-create-a-devon4quarkus-app.asciidoc

```diff
@@ -0,0 +1,224 @@
+* An installation of the *Visual Studio 2017 Visual C++ Build Tools* is needed. You can load it from https://aka.ms/vs/15/release/vs_buildtools.exe[here].
+* The creation of the native executable is only able in the x64 version of the *Native Tool Command Prompt for VS 2017*. Start the command prompt and jump into the *getting-started* folder. Build the native executable with the command `mvn package -Pnative` and a target folder with the executable *getting-started-1.0.0-SNAPSHOT-runner.exe* will be created. Start the application and you should see the fast startup time and if you inspect the process properties also the low memory usage. You can again access http://localhost/hello to check the application.
+
+
+. MacOS
+For MacOS just follow this guide https://quarkus.io/guides/building-native-image#prerequisites-for-oracle-graalvm-ceee.
+
+
+==== Build native executables without GraalVM 
+
+You can build the Linux executables without installing GraalVM or sometimes you just need the Linux executables on Windows to run them in a container. To do that you need a working container runtime, we use Docker in this guide. You can install Docker with your devonfw-ide distribution just follow this description https://github.com/devonfw/ide/blob/master/documentation/docker.asciidoc[Docker with devonfw-ide]. 
+
+[sidebar]
+.Docker on Windows
+--
+There are two modes for Docker on Windows, if you are using it in Hyper-V mode and not WSL 2 mode you have to share the project drive in the Docker settings or you will face some errors. 
+--
+We can start building our container image for the project.
+Create an executable in a container runtime with the command `mvn package -Pnative "-Dquarkus.container-image.build=true"` normally quarkus automatically detects the container runtime, but sometimes an error occurs and then you should try to you can explicitly select the container runtime with the parameter `"-Dquarkus.native.container-runtime=docker"´. You can omit the `-Pnative` parameter to create a Dockerfile with the .jar and not the native executable.
+
+Another way to create the native application without using the command prompt is the Dev UI we already showed. Start the application and go to http://localhost:8080/q/dev/. Select Build in the tile named Container Image and you are able to select a *Build Type* and a *Builder Type* and build a selected .jar oder native executable in the background.
+
+== Conclusion
+
+
+This was a first glance over the functionality of devon4quarkus. If something is missing in this overview or you need more information follow our following guides and some in-depth going guides can be found at https://quarkus.io/guides/[Quarkus Guides] and the Katacoda tutorials by https://www.katacoda.com/openshift/courses/developing-with-quarkus[OpenShift for Quarkus Developement]. 
```

Comment:
So far we have no functionality in devon4quarkus. It is just simple quarkus stuff. So we need to distinguish these two terms